webui/webui.go

```diff
@@ -196,23 +238,27 @@ func (h handler) listHandler(w http.ResponseWriter, r *http.Request, l zerolog.L
 		l.Error().Err(err).Str("page", "/list").Msg("cannot list namespaces")
 	}
 
-	var nsList ListNamespacesAndStates
+	p := Page{
+		Version:   h.version,
+		BuildDate: h.builddate,
+	}
+	// var nsList ListNamespacesAndStates
 	for _, n := range namespaces.Items {
 		if a, ok := n.Annotations[h.prefix+engine.ControllerName]; ok && a == h.controllerName {
 			val := n.Annotations[h.prefix+engine.DesiredState]
 			ns := Namespace{Name: n.Name}
 			switch val {
 			case engine.Suspended:
-				ns.State = "❌"
+				ns.State = "🔴"
```

Comment:
👍🏻  good call for changing ❌ into another form because it is often interpreted as an error from a user's point of view.

💭 In the same manner, I think that anything "RED" is also understood as an error.
Why not use another color with a "lower" level of criticity interpretation.

⚪  ? 🟠  ? 🟡  ?